customersupport/templates/new-call.html

```diff
@@ -66,12 +67,13 @@
             <div class="col-sm-6">
                 <div class="form-group">
                     <div class="input-group">
-                        <select class="form-control" id="type">
+                        <select class="form-control" id="ticketType">
                             <option>Refund</option>
                             <option>Replace</option>
                         </select>
                         <span class="input-group-btn">
-                  <a class="btn btn-primary" href="new-ticket.html" role="button">New Ticket</a>
+                            <button type="button" class="btn btn-primary" id="createTicket" data-toggle="modal"
```

Comment:
We need to handle what happens when there is no order in the order field and this button is clicked. Take a look at @Daniel-J-Fisher's "List Customer Tickets" example to see how he did it, but he disabled the button until there was a customer filled it.